src/__tests__/server.test.ts

```diff
@@ -0,0 +1,22 @@
+import request from 'supertest';
+import Server from '../modules/server';
+
+const server = new Server();
+
+it('test GET method in \'/hello\' endpoint', async () => {
+  server.startServer();
+
+  const response = await request((await server.getApp()).callback()).get('/hello');
```

Comment:
How about changing supertest to[http-client](https://www.npmjs.com/package/typescript-http-client) ?

Reply:
I think that supertest is more sutable for this kind of test, because it tests our app independely of wich URL and port the app is running. It's common to use supertest to endpoint testing, along side with jest.
References:
- https://dev.to/nedsoft/testing-nodejs-express-api-with-jest-and-supertest-1km6
- https://medium.com/better-programming/adding-tests-to-express-apps-with-jest-and-supertest-d0e7aaac4c08
- https://www.albertgao.xyz/2017/05/24/how-to-test-expressjs-with-jest-and-supertest/